stock_picking_backorder_strategy/models/stock_picking.py

```diff
@@ -12,7 +12,7 @@ def _check_backorder(self):
         # If strategy == 'manual', let the normal process going on
         if self.picking_type_id.backorder_strategy == "manual":
             return super(StockPicking, self)._check_backorder()
-        return False
+        return self.browse()
```

Comment:
This method should be re-implemented as the standard one is able to check multiple pickings ([link](https://github.com/odoo/odoo/blob/3711c299523c8da4abb6066e505ec05090d1567d/addons/stock/models/stock_picking.py#L1017-L1041)), here the `self.ensure_one()` is breaking this.

Reply:
@sebalix This should have been fixed, I did the review myself and saw the code. It must never have been pushed. Anyway RujiaL won't be working on Odoo anymore as she left so make a PR and I can review.

Reply:
@sebalix I don't know where you got your snippet, it is not the current code

Reply:
I don't know myself! Sorry for the noise.